azkaban-common/src/main/java/azkaban/executor/container/ContainerizedImpl.java

```diff
@@ -20,4 +20,5 @@
 public interface ContainerizedImpl {
   void createContainer(final int executionId) throws ExecutorManagerException;
   void deleteContainer(final int executionId) throws ExecutorManagerException;
+  void logPodDetails(final int execId) throws ExecutorManagerException ;
```

Comment:
Currently this doesn't seem to be invoked anywhere beyond `KubernetesContainerizedImpl`, is that intentional ?
(we could get away w/o adding it to the interface then)

Reply:
I think long term we should be logging our PodSpec.yaml in logger.debug mode. This will enable a kubectl apply -f of the yaml which will help in debugging. Not sure where that needs to be added. I can remove it here, and from the interface and the Mock if you think that should be a future consideration.